### PR TITLE
Adding the sinner's mask

### PR DESCRIPTION
**Pull request by Agoichi**
## About the PR
New mask, blocks identity, nothing more

## Why / Balance
We need it. Mostly for the guardians who'll be executioner

## Media

image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: The Sinner's Mask

### CODE REVIEW

**Comment by Agoichi** [comment]
Что не так блять, ЧТО НЕ ТАК-ТО БЛЯТЬ ЕБАННЫЙ УБУНТУ

File: Resources/Prototypes/_CP14/Entities/Clothing/Masks/pluto-mask.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- type: entity
+  parent: ClothingMaskBase
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Clothing/Masks/pluto-mask.yml:2]: Add CPBaseMask, with equip time and unequip time

File: Resources/Prototypes/_CP14/Entities/Clothing/Masks/pluto-mask.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  id: CPSinnerMask
+  name: Маска Грешника
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Clothing/Masks/pluto-mask.yml:4]: Eng name and description 
+ Localization in separate file

**Review by TheShuEd — CHANGES_REQUESTED** [review]

**Comment by Agoichi** [comment]
Done

**Comment by Agoichi** [comment]
блять

**Review by TheShuEd — APPROVED** [review]

**Comment by Agoichi** [comment]
Тест Убунбу для дуракав

**Comment by Deserty0** [comment]
ты бы со всей локализацией сделал так

**Pull request merged**